lib/www/common.php

```diff
@@ -253,6 +254,37 @@ function putSubmissions($cdatas, $restrictions, $limit = 0, $highlight = null)
 				}
 			}
 			echo "</td>";
+			if ( $testcases ) {
+				$judgingid = $row['judgingid'];
+				$probid = $row['probid'];
+				$runs = $DB->q('SELECT r.runresult, t.rank
+						FROM testcase t
+						LEFT JOIN judging_run r ON ( r.testcaseid = t.testcaseid
+							AND r.judgingid = %i )
+						WHERE t.probid = %i ORDER BY rank',
+						$judgingid, $probid);
+				$runinfo = $runs->gettable();
```

Comment:
you could use $DB->q('TABLE SELECT' above instead of SELECT + gettable (a feature of our db lib)